nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/PutS3Object.java

```diff
@@ -132,6 +132,19 @@
         .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
         .build();
 
+    public static final PropertyDescriptor CONTENT_TYPE = new PropertyDescriptor.Builder()
+        .name("Content Type")
```

Comment:
Could you use the combination of ``.name()`` and ``.displayName()``? name ensures backward compatibility when displayName allows to be more flexible on what is displayed to users. We are trying to ensure every new processor properties is using both.